Documentation/docs/learn/fda_sw_development_guidelines.md

```diff
@@ -1,4 +1,6 @@
-# FDA Guidelines for Off-the-Shelf software
+# FDA guidelines for software development
+
+## FDA guidelines for off-the-shelf software
```

Comment:
Thanks Jon. While I agree about using sentence style case for section headers, I would prefer _title case_ for the title. Just like in the original wiki page (https://itk.org/Wiki/FDA_Guidelines_for_Software_Development):

    FDA Guidelines for Software Development

Reply:
Sorry Niels, but I am being as consistent as possible with the rest of the headings.

Reply:
Thanks for your reply, but sorry, but I still prefer title case for titles. The _Python Quick Start_ also uses title case:
 
https://github.com/InsightSoftwareConsortium/ITK/blob/4543ba6dce61bdc4540bb0641b2aafae10056b9a/Documentation/docs/learn/python_quick_start.md?plain=1#L1

So why would we not using title case for titles, consistently?

Reply:
Niels, I did not contribute to that file, and I said that in the files I contributed I was being as consistent _as possible_. Please do change all of them in a separate PR if you feel like they should be, but this PR does address your original request.

Reply:
Thanks for addressing my request, Jon. I understand now, you just want to be consistent with your previous contributions. So don't you _like_ title case at all?

Anyway, it's no big deal, I won't press the "Request changes" button.

Reply:
> So don't you like title case at all?

No, I don't. But as said, feel free to open a PR that changes them all.

Reply:
Thanks for your clarification!